packages/go/cypher/models/pgsql/translate/expression.go

```diff
@@ -542,10 +545,55 @@ func (s *ExpressionTreeTranslator) PopBinaryExpression(operator pgsql.Operator)
 	}
 }
 
-func (s *ExpressionTreeTranslator) PopPushBinaryExpression(operator pgsql.Operator) error {
+func (s *ExpressionTreeTranslator) PopPushBinaryExpression(scope *Scope, operator pgsql.Operator) error {
```

Comment:
This is a pretty beefy fn that might benefit from breaking down into smaller chunks per major switch case perhaps?

Reply:
Noted. I'd like to come back to this in another changeset.